### PR TITLE
[pre-existing] Fix pre-populate for country and state

### PR DESCRIPTION
**Pull request by juanfra**
https://central.tri.be/issues/63915 after QA

Fix pre-populate for country and state. Was a pre-existent.

### CODE REVIEW

File: src/admin-views/create-venue-fields.php [hunk 1]
```diff
@@ -153,7 +155,7 @@ class="tribe-dropdown"
 			<option value=""><?php esc_html_e( 'Select a State:', 'the-events-calendar' ); ?></option>
 			<?php
 			foreach ( Tribe__View_Helpers::loadStates() as $abbr => $fullname ) {
-				echo '<option value="' . esc_attr( $abbr ) . '">' . esc_html( $fullname ) . '</option>';
+				echo '<option value="' . esc_attr( $abbr ) . '" ' . selected( ( $_VenueState === $abbr || $_VenueState === $fullname ), true, false ) . '>' . esc_html( $fullname ) . '</option>';
```
Comment (bordoni) [inline src/admin-views/create-venue-fields.php]: Actually move the logic out of the template.

File: src/admin-views/venue-meta-box.php [hunk 2]
```diff
@@ -123,10 +122,9 @@ class="tribe-dropdown"
 			<option value=""><?php esc_html_e( 'Select a State:', 'the-events-calendar' ); ?></option>
 			<?php
 			foreach ( Tribe__View_Helpers::loadStates() as $abbr => $fullname ) {
-				$state = -1 !== $_VenueStateProvince ? $_VenueStateProvince : $currentState;
 				// support matching by state abbreviation OR fullname.
 				// NOTE: converts to abbreviation on save
-				echo '<option value="' . esc_attr( $abbr ) . '"' . selected( ( $state === $abbr || $state === $fullname ), true, false ) . '>' . esc_html( $fullname ) . '</option>';
+				echo '<option value="' . esc_attr( $abbr ) . '"' . selected( ( $currentState === $abbr || $currentState === $fullname ), true, false ) . '>' . esc_html( $fullname ) . '</option>';
```
Comment (bordoni) [inline src/admin-views/venue-meta-box.php]: Move logic out of template.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$selected' | [create-venue-fields.php:159](https://github.com/moderntribe/the-events-calendar/blob/9c9fa64ced9edabca2b4e8887deccf1ff817ab38/src/admin-views/create-venue-fields.php#L159)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$selected' | [venue-meta-box.php:128](https://github.com/moderntribe/the-events-calendar/blob/9c9fa64ced9edabca2b4e8887deccf1ff817ab38/src/admin-views/venue-meta-box.php#L128)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$selected' | [create-venue-fields.php:159](https://github.com/moderntribe/the-events-calendar/blob/f3c4e8e51dd6499455e9bd04f8751c4ac74c52e6/src/admin-views/create-venue-fields.php#L159)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$selected' | [venue-meta-box.php:128](https://github.com/moderntribe/the-events-calendar/blob/f3c4e8e51dd6499455e9bd04f8751c4ac74c52e6/src/admin-views/venue-meta-box.php#L128)

> via jenkins/codesniffer-phpcs

**Pull request merged**